src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -687,11 +687,32 @@ public boolean muteParticipant(ColibriConferenceIQ channelsInfo, boolean mute, M
         }
 
         request.addContent(requestContent);
-        UtilKt.tryToSendStanza(connection, request);
+        try
+        {
+            IQ response = UtilKt.sendIqAndGetResponse(connection, request);
```

Comment:
This makes `muteParticipant` blocking. I can make the mute IQ handler async if you want.

Reply:
Well, I check that the iq handlers are executed async in a smacks cached thread pool.